mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -21,6 +21,7 @@ import scala.annotation.varargs
 
 import org.apache.spark.annotation.Since
 import org.apache.spark.api.java.{JavaDoubleRDD, JavaRDD}
+import org.apache.spark.ml.stat._
```

Comment:
I guess it's not reasonable to move the implementation to /mllib ? No big deal, just wondering if we can generally have ml call mllib and not the reverse, but this isn't 100% true even now.

Reply:
I guess I can move impl to /ml, will have a try

> Most of the change here is breaking out the inner class right?

Yes, if we move impl to /ml. I guess I may not need to breaking out it.

Reply:
@srowen I guess we can not easily move the implementation to /mllib, since Summizer is impled in /ml.